source/frequently-asked-questions/frequently-asked-questions.rst

```diff
@@ -2,3 +2,333 @@
+-  `Where can I find the main project
+   repositories? <#where-can-i-find-the-main-project-repositories>`__
+-  `Where can I learn about the Ether sale and
+   mining? <#where-can-i-learn-about-the-ether-sale-and-mining>`__
+-  `Clients <#clients>`__
+-  `Where can I find official
+   releases? <#where-can-i-find-official-releases>`__
+-  `How to install development
+   builds? <#how-to-install-development-builds>`__
+-  `How to install the clients from
+   source? <#how-to-install-the-clients-from-source>`__
+-  `Mining <#mining>`__
+-  `How can I mine Ether? <#how-can-i-mine-ether>`__
+-  `How to get free testnet Ether? <#how-to-get-free-testnet-ether>`__
+-  `Contracts <#contracts>`__
+-  `Where can I learn about contract
+   development? <#where-can-i-learn-about-contract-development>`__
+-  `Where can I learn Solidity? <#where-can-i-learn-solidity>`__
+-  `Where can I learn Serpent? <#where-can-i-learn-serpent>`__
+-  `Where can I learn LLL? <#where-can-i-learn-lll>`__
+-  `Where can I learn Solidity? <#where-can-i-learn-solidity>`__
+-  `How to test contracts? <#how-to-test-contracts>`__
+-  `How to deploy contracts
+   automatically? <#how-to-deploy-contracts-automatically>`__
+-  `Where to find example
+   contracts? <#where-to-find-example-contracts>`__
+-  `ÐApps <#Ðapps>`__
+-  `Where can I learn about the Ethereum
+   APIs? <#where-can-i-learn-about-the-ethereum-apis>`__
+-  `Where can I learn about ÐApp
+   development? <#where-can-i-learn-about-Ðapp-development>`__
+-  `Where can I find ÐApp development
+   tools? <#where-can-i-find-Ðapp-development-tools>`__
+-  `Where can I find example ÐApps? <#where-can-i-find-example-Ðapps>`__
+-  `IRC <#irc>`__
+-  `How can I join the Ethereum IRC
+   channels? <#how-can-i-join-the-ethereum-irc-channels>`__
+-  `Where can I find the Ethereum IRC
+   logs? <#where-can-i-find-the-ethereum-irc-logs>`__
+-  `Where can I learn about the ZeroGox
+   bot? <#where-can-i-learn-about-the-zerogox-bot>`__
+-  `More Questions and Answers <#more-questions-and-answers>`__
+
+.. raw:: html
+
+   <!-- toc stop -->
+
+Ethereum
+--------
+
+What is Ethereum?
+~~~~~~~~~~~~~~~~~
+
+There are several ways to answer this question, which are introduced on
+the dedicated `What is
+Ethereum <http://github.com/ethereum/wiki/wiki/What-is-Ethereum>`__ wiki
+page.
+
+If you prefer to learn by watching videos, see: 
+
++ `Introducing Ethereum <http://youtu.be/mRWNr3MA2jo>`__ (Video, 10min) 
++ `Vitalik Buterin reveals Ethereum at Bitcoin Miami 2014 <http://youtu.be/l9dpjN3Mwps>`__ (Video, 28min) 
++ `Singularity 1 on 1: Ethereum is a Decentralized Consensus Platform <http://youtu.be/fbEtivJIfIU>`__ (Video, 69min) 
++ `Our second Reddit "Ask Me Anything" for community selected questions <http://www.reddit.com/r/IAmA/comments/2bjmgb/hi_we_are_the_ethereum_project_team_ask_us/>`__ (not actually a video)
+
+Where can I learn more about Ethereum?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
this should be a cross ref to community section of the ecosystem. 
Forum will cease to exist. Gitter channels should be listed in detail 
based on https://gitter.im/orgs/ethereum/rooms